cmd/run/run.go

```diff
@@ -459,23 +471,15 @@ func RunServer(ctx context.Context, config *Config) error {
 		return fmt.Errorf("failed to initialize authenticator: %w", err)
 	}
 
-	unaryServerInterceptors := []grpc.UnaryServerInterceptor{
```

Comment:
You're gonna hate me, but do you mind if we save this for a different PR? I'd like to separate the code reveiw for this change from the scope of work for the tracing stuff.

I have some comments on this stuff, but don't want it to hinder progress on the tracing stuff.

Reply:
haha. No worries. Reverted... mostly.